test/ui/cypress/e2e/deployment_driver.cy.js

```diff
@@ -18,38 +11,101 @@ describe('Deployment driver web UI', () => {
     cy.get('@main').contains('Deployment Steps')
     cy.get('@main').get('div.deployProgress').as('statusArea')
     cy.get('@statusArea').get('button.cancelButton').contains('Cancel Deployment')
+
+    cy.get('.pf-c-brand').should('be.visible')
+    cy.get('.pf-c-alert__title')
+      .contains('You currently have a subscription to Ansible Automation Platform')
+    cy.get('p').contains('To manage or setup new subscription, visit the')
+    cy.get('p > .pf-c-button')
+      .contains('Red Hat Hybrid Cloud Console')
+      .should('have.attr', 'href', 'https://console.redhat.com/')
+
+    // Check the H1 title
+    cy.get('h1')
+      .contains('Ansible Automation Platform Deployment Engine')
+      .should('be.visible') 
   })
 
   it('left navigation has expected navigation links', () => {
+    cy.get('#nav-toggle').click()
```

Comment:
Why does the navigation need to be clicked? A better approach would be to check its visibility and click on it if it is hidden.

Reply:
@tznamena , I am not sure why the navigation menu wasn't displayed by default when I used cypress to run Deployment Engine UI automation. Without the `click()`, the navigation test would fail.

Reply:
The reason was most likely that cypress used some very small viewport. With an explicitly configured viewport we can be sure this is always displayed.

Reply:
I will test if this test case works without the click() with cy.viewport(1920,1080) setting.